.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
i believe that this would only for github-actions workflow based on https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem 

In case you want to enable it for go.mod dependencies you might need to change the [package-ecosystem](https://github.com/weaveworks/pipeline-controller/pull/190/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28)